src/components/PeoplePicker/PeoplePicker.scss

```diff
@@ -116,7 +131,7 @@ $personaItemHeight: 42px;
   border: 1px solid $ms-color-neutralTertiaryAlt;
   display: none;
   margin-bottom: -1px;
-  max-width: 340px;
+  min-width: 340px;
```

Comment:
Hmm... this seems bad. It will add a scrollbar on mobile devices @320px wide, which is a pretty common size. If we're doing this, do we need `-width` properties at all?